clang/docs/checkedc/3C/README.md

```diff
@@ -23,22 +23,24 @@ some point.
 
 ## What 3C users should know about the development process
 
-Development of 3C is led by [Correct Computation,
-Inc.](https://correctcomputation.com/) (CCI) in the
+Checked C development as a whole is led by Microsoft in the
+https://github.com/microsoft/checkedc-clang repository. 3C is included
+in the Checked C codebase, but 3C development is led by [Correct
+Computation, Inc.](https://correctcomputation.com/) (CCI) in the
 https://github.com/correctcomputation/checkedc-clang repository, which
-is a fork of Microsoft's Checked C repository
-(https://github.com/microsoft/checkedc-clang). Issues and pull
-requests related to 3C should be submitted to the CCI repository; see
-[CONTRIBUTING.md](CONTRIBUTING.md) for more information. Changes are
-merged frequently from Microsoft's repository to CCI's and less
-frequently in the opposite direction. While some automated tests of 3C
-are run in Microsoft's repository, the coverage of these tests is
-currently mediocre, so changes to Microsoft's repository may break
-some functionality of its copy of 3C. Thus, users of 3C can choose
-either CCI's repository (for the latest 3C with a somewhat older
-Checked C) or Microsoft's (for the latest Checked C with 3C that is
-significantly older and possibly broken). This workflow may change in
-the future.
+is a fork of Microsoft's repository; changes are periodically merged
+back and forth. (That is, CCI plays roughly the role of a ["subsystem
+maintainer"](https://www.kernel.org/doc/html/latest/process/2.Process.html#how-patches-get-into-the-kernel)
+for 3C, for those familiar with that term from Linux kernel
+development.) Both 3C and the rest of the Checked C tools (including
+the compiler) are available in both repositories, but the repositories
+will generally have different versions of different parts of the code
+at any given time, so [you'll have to decide which repository is more
+suitable for your work](#which-checkedc-clang-repository-to-use).
+
+Issues and pull requests related to 3C should be submitted to CCI's
+repository; see [CONTRIBUTING.md](CONTRIBUTING.md) for more
+information.
 
 As of November 2020, 3C is pre-alpha quality and we are just starting
```

Comment:
Change this to be "As of March 2021" ... ?

Reply:
Done.